spring/boot3-actuator-autoconfigure/src/test/resources/application-autoConfTest.yml

```diff
@@ -13,5 +13,7 @@ management:
   endpoints:
     web:
       exposure:
-        include: health, heapdump, loggers, prometheus
+        include: health, heapdump, loggers, prometheus, info
```

Comment:
How about also adding `exclude` and seeing if it works just in case?
We can probably exclude `health` which is added by default.

Reply:
good catch.
fixed: https://github.com/line/armeria/pull/4880/commits/1a50b175af865b66670bcda04711e65f0f365b0b

ref:
https://github.com/spring-projects/spring-boot/blob/e3aac5913ed3caf53b34eb7750138a4ed6839549/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/expose/IncludeExcludeEndpointFilter.java#L123-L139